vllm/envs.py

```diff
@@ -37,6 +37,7 @@
     VLLM_FUSED_MOE_CHUNK_SIZE: int = 64 * 1024
     VLLM_USE_RAY_SPMD_WORKER: bool = False
     VLLM_USE_RAY_COMPILED_DAG: bool = False
+    VLLM_USE_RAY_COMPILED_DAG_NCCL: bool = True
```

Comment:
It might make more sense as `VLLM_RAY_COMPILED_DAG_COMM_BACKEND=VLLM`

Also should ensure it shows up in docs: https://docs.vllm.ai/en/latest/serving/env_vars.html

Reply:
I renamed it to VLLM_USE_RAY_COMPILED_DAG_NCCL_CHANNEL as suggested by Sang. Let me know how it sounds to you.

Reply:
That sounds fine as well